lib/amqp/session.rb

```diff
@@ -130,11 +130,6 @@ def logger
       end
 
       def logger=(logger)
```

Comment:
It's set as an attr_accessor for the instance, and these methods are defined at the class level.